sparkbench/pom.xml

```diff
@@ -121,6 +128,15 @@
         <spark.version>2.0.0</spark.version>
         <spark.bin.version>2.0</spark.bin.version>
       </properties>
+      <modules>
+        <module>micro</module>
+        <module>ml</module>
+        <module>websearch</module>
+        <module>graph</module>
+        <module>sql</module>
+        <module>streaming</module>
+        <module>structuredStreaming</module>
+      </modules>
```

Comment:
Why add these lines? This will build all modules even if `-module` is specified.